README.md

```diff
@@ -347,6 +367,22 @@ The **plasmidfinder.tsv** output file generated by `staramr` contains the follow
 | SRR1952908 | ColpVC    | 98.96     | 100      | 193/193                 | contig00038 | 1618  | 1426  | JX133088  |
 | SRR1952908 | IncFIB(S) | 98.91     | 100      | 643/643                 | contig00024 | 10302 | 9660  | FN432031  |
 
+## mlst.tsv
+
+The **mlst.tsv** output file generated by `staramr` contains the following columns:
+
+* __Isolate ID__: The id of the isolate/genome file(s) passed to `staramr`.
+* __Scheme__: The organism that `MLST` has identified.
```

Comment:
Could you change **organism** to **scheme** (it's possible for there to be multiple schemes for a particular organism, in particular, ecoli has two schemes *ecoli* and *ecoli_2*).

Reply:
Fixed in my next commit